scala-package/core/src/main/scala/org/apache/mxnet/Shape.scala

```diff
@@ -56,4 +59,5 @@ class Shape(dims: Traversable[Int]) extends Serializable {
 object Shape {
   def apply(dims: Int *): Shape = new Shape(dims: _*)
   def apply(dims: Traversable[Int]): Shape = new Shape(dims)
+  @varargs def create(dims: Int*): Shape = new Shape(dims)
```

Comment:
please mark it experimental

Reply:
no experimental is for unstable apis which can cause problem